transport/src/main/java/io/netty/channel/ChannelHandler.java

```diff
@@ -194,52 +194,26 @@ default void handlerRemoved(ChannelHandlerContext ctx) throws Exception {
         // no value
     }
 
-    /**
-     * Indicates that the annotated event handler method in {@link ChannelHandler} will not be invoked by
-     * {@link ChannelPipeline}. This annotation is only useful when your handler method implementation
-     * only passes the event through to the next handler, like the following:
-     *
-     * <pre>
-     * {@code @Skip}
-     * {@code @Override}
-     * public void channelActive({@link ChannelHandlerContext} ctx) {
-     *     ctx.fireChannelActive(); // do nothing but passing through to the next handler
-     * }
-     * </pre>
-     *
-     * <p>
-     * Note that this annotation is not {@linkplain Inherited inherited}.  If you override a method annotated with
-     * {@link Skip}, it will not be skipped anymore.  Similarly, you can override a method not annotated with
-     * {@link Skip} and simply pass the event through to the next handler, which reverses the behavior of the
-     * supertype.
-     * </p>
-     */
-    @Target(ElementType.METHOD)
-    @Retention(RetentionPolicy.RUNTIME)
-    @interface Skip {
-        // no value
-    }
-
     /**
      * The {@link Channel} of the {@link ChannelHandlerContext} was registered with its {@link EventLoop}
      */
-    @Skip
+    @ChannelHandlerMask.Skip
```

Comment:
any reason not to import `ChannelHandlerMask.Skip`?

Reply:
nope... I don't cary . If you prefer I can do for sure.

Reply:
was just a minor curiosity if it was for readability or something, just looks a bit cleaner without it imo... I see you already changed anyhow, thanks !

Reply:
@njhill yep... I try to keep people happy 🤣